test/parallel/test-crypto-aes-128-ecb.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
```

Comment:
Please either rename this file to better match what it tests, or add the test case to an existing test file that tests key objects.

Also, please add a comment explaining that this is a regression test for https://github.com/nodejs/node/issues/40814.